tests/test_client.py

```diff
@@ -261,3 +261,16 @@ def test_device_proxy_dir_method(tango_test):
     assert set(methods) <= set(lst)
     # Check internals
     assert set(internals) <= set(lst)
+
+
+def test_device_polling_command(tango_test):
+
+    dct = {"SwitchStates": 1000, "DevVoid": 10000, "DumpExecutionState": 5000}
+
+    for command in dct.items():
+        tango_test.poll_command(command[0], command[1])
```

Comment:
You can use unpacking here instead of indexing:

    for command, period in dct.items():
        tango_test.poll_command(command, period)